src/GreenFunc.jl

```diff
@@ -3,8 +3,11 @@ using StaticArrays, Lehmann, CompositeGrids
 # Write your package code here.
 
 
+
 include("green/Green.jl")
+include("green/GreenSym.jl")
 export TimeDomain, ImTime, ReTime, ImFreq, ReFreq, DLRFreq
```

Comment:
put include and export of one file together so readers know which is exported from which file:

include("fname1")
export F1, ...

include("fname2")
export F2, ...